FSQLocationBroker/FSQLocationBroker.m

```diff
@@ -516,13 +516,13 @@ - (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray
     BOOL isBackgrounded = applicationIsBackgrounded();
     CLLocation *mostRecentLocation = [locations firstObject];
     for (CLLocation *location in locations) {
-        if ([mostRecentLocation.timestamp laterDate:location.timestamp] == location.timestamp) {
+        if ([mostRecentLocation.timestamp compare:location.timestamp] == NSOrderedAscending) {
```

Comment:
OOooooOOOOh - I hadn't expanded the context to realize that 517-522 are just finding the most recent location from `locations` - yes then nevermind, the check would be unnecessary anyways because we'll never get into the for loop if `locations.count == 0`